### PR TITLE
facility to fhir

### LINKED ISSUE #4799: Create FHIR Organization using facility information
### Background
To send FHIR events to the Universal Pipeline, first we'll need to convert our TestEvents to the FHIR format.

### Action requested
Create a toFHIR method for Facility; it should return an [Organization FHIR model](https://www.hl7.org/fhir/organization.html).

### Acceptance Criteria
* Facilities can be converted into an Organization FHIR object
   * Should include identifier, type, name, telecom, address
* Facilities should use information as needed from the parent Organization to fill out necessary fields.

### Additional context
An example organization can be found [here](https://gist.github.com/BobanL/dfe9bfd37f62c6032a1df866fdad133b#file-reportstream-correction-fhir-L308-L363) or in the [report stream fhir examples](https://github.com/CDCgov/prime-reportstream/tree/fa3db4c1e1025d583df06d317fb2037ab1e44dff/prime-router/src/testIntegration/resources/datatests).

### PR DESCRIPTION
**Pull request by BobanL**
# BACKEND PULL REQUEST

## Related Issue

- Resolves #4799 

## Changes Proposed

- Convert facility to organization 

## Additional Information

- We are using facility rather than organization since that is the actual place that the test is taking place at.

## Testing

- Unit tests pass

### CODE REVIEW

File: backend/src/test/java/gov/cdc/usds/simplereport/db/model/FacilityTest.java [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    var actual = facility.toFhir();
+
+    FhirContext ctx = FhirContext.forR4();
+    IParser parser = ctx.newJsonParser();
+
+    String actualSerialized = parser.encodeResourceToString(actual);
+    var expectedSerialized =
+        IOUtils.toString(
+            Objects.requireNonNull(
+                PersonTest.class.getClassLoader().getResourceAsStream("fhir/organization.json")),
```
Comment (georgehager) [inline backend/src/test/java/gov/cdc/usds/simplereport/db/model/FacilityTest.java:48]: It's a little confusing to me that this calls the classLoader on the PersonTest class - what is the reason for that?

**Review by georgehager — APPROVED** [review]
lgtm! (given my limited knowledge of fhir)

**Review by georgehager — APPROVED** [review]
😁

**Review by emyl3 — APPROVED** [review]
also LGTM based on my limited understanding of FHIR... newly added unit tests passed locally ✅ Thank you, Boban! 😸

**Pull request merged**